test-harness/gold-master-files/__path____pets_123___method___POST___headers____content-type___application_vnd.api+json____body____\_hello\__\_world\____.json

```diff
@@ -0,0 +1,32 @@
+{
+  "request": {
+    "body": "{\"hello\":\"world\"}",
+    "method": "POST",
+    "headers": {
+      "content-type": "application/vnd.api+json"
+    },
+    "path": "/pets/123",
+    "host": "http://localhost:4010"
+  },
+  "response": {
+    "status": 500,
+    "statusText": "Internal Server Error",
```

Comment:
Hmm wait, I'm not sure I follow. Is that the expected response? You sure we want 5xx?
Not sure if this help is useful. Maybe it should at least reply with a correct 200?

Reply:
yeah that is confusing, I'm open to better ideas.

Basically the issue this PR is fixing was at Fastify level, where the framework was refusing the request, completely. The fact that you got a message error from Prism means that the request went through correctly. Does that clarify?

Reply:
Actually I have a better idea, hold on